source/docs/casper/developers/prerequisites.md

```diff
@@ -11,64 +13,327 @@ This section covers:
+Follow the steps below to install necessary software for the development environment.
+
+## Install the environment on macOS / Linux
+
+## Step 1.
+
+### Install xcode command line tools on macOS {#install-xcode}
+
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+xcode-select --install
+```
+
+</TabItem>
+</Tabs>
+
+Check the version 
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+xcode-select -p
+```
+
+</TabItem>
+</Tabs>
+
+### Get curl on Linux {#install-curl}
+
+<Tabs>
+<TabItem value="Linux" label="Linux">
+
+```bash
+sudo apt install curl
+```
+</TabItem>
+</Tabs>
+
+## Step 2.
+
+### Install rust {#install-rust}
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+</TabItem>
+<TabItem value="Linux" label="Linux">
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+</TabItem>
+</Tabs>
+
+Check the rustup version
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+rustup --version
+```
+
+</TabItem>
+<TabItem value="Linux" label="Linux">
+
+```bash
+rustup --version
+```
+
+</TabItem>
+</Tabs>
+
+The installation script automatically adds Rust to your system PATH after your next login.
+To start using Rust right away instead of restarting your terminal, run the following command in your shell to add Rust to your system PATH manually:
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+source $HOME/.cargo/env
+```
+
+</TabItem>
+<TabItem value="Linux" label="Linux">
+
+```bash
+source $HOME/.cargo/env
+```
+
+</TabItem>
+</Tabs>
+
+## Step 3.
+
+### Install brew on macOS {#install-brew}
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+</TabItem>
+</Tabs>
+
+### Install additional packages to be able to install cargo-casper and casper-client {#install-adds}
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+brew install pkg-config
+brew install openssl 
+```
+
+</TabItem>
+<TabItem value="Linux" label="Linux">
+
+```bash
+sudo apt-get install pkg-config
+sudo apt-get install openssl
+sudo apt-get install libssl-dev
+```
+
+</TabItem>
+</Tabs>
+
+### Install cargo on Linux {#install-linux-cargo}
+
+<Tabs>
+<TabItem value="Linux" label="Linux">
+
+```bash
+sudo apt install cargo
+```
+
+</TabItem>
+</Tabs>
+
+## Step 4.
+
+### Install the cargo-casper {#install-cargo-casper}
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+cargo install cargo-casper
+```
+
+</TabItem>
+<TabItem value="Linux" label="Linux">
+
+```bash
+cargo install cargo-casper
+```
+
+</TabItem>
+</Tabs>
+
+Check the version
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+cargo-casper --version
```

Comment:
Maybe add expected output? Small optimization. But "check the version" sounds like the version needs to be something specific. what we're doing is just verifying that it is installed. Maybe it's enough to say "verify the installation by..."